packages/frontend/src/components/LightdashVisualization/VisualizationProvider.tsx

```diff
@@ -51,6 +51,7 @@ type VisualizationContext = {
     isSqlRunner: boolean;
     dimensions: Dimension[];
     metrics: Metric[];
+    allMetrics: (Metric | TableCalculation)[];
```

Comment:
what's the difference between this and `metrics` and `tableCalculations` ? 

Is it different to `allMetrics = [...metrics, ...tableCalculations]` ? If yes it should be named differently

Reply:
im not sure what to name this tbh. we keep using name metrics for metrics, additional metrics and table calculations...